opencti-platform/opencti-front/src/components/FilterIconButton.tsx

```diff
@@ -23,9 +23,11 @@ interface FilterIconButtonProps {
   availableRelationFilterTypes?: Record<string, string[]>;
 }
 
-const FilterIconButton: FunctionComponent<FilterIconButtonProps> = ({
-  availableFilterKeys,
-  filters = emptyFilterGroup,
+interface FilterIconButtonIfFiltersProps extends FilterIconButtonProps {
```

Comment:
I'm not sure about this name 🤔

What about `ActiveFilterIconButton` ?

Reply:
it's not really the active filters, the inactive ones are also in the component..